api/src/main/resources/workbench-api.yaml

```diff
@@ -9504,3 +9506,175 @@ definitions:
+      - ASIAN_FILIPINO
+      - ASIAN_HMONG
+      - ASIAN_JAPANESE
+      - ASIAN_KOREAN
+      - ASIAN_LAO
+      - ASIAN_PAKISTANI
+      - ASIAN_VIETNAMESE
+      - ASIAN_OTHER           # Asian / None of these fully describe me, and I want to specify
+      - BLACK                 # Black, African American, or of African descent
+      - HISPANIC              # Hispanic, Latino, or Spanish descent
+      - MENA                  # Middle Eastern or North African
+      - NHPI                  # Native Hawaiian or other Pacific Islander
+      - WHITE                 # White, or of European descent
+      - OTHER                 # None of these fully describe me, and I want to specify
+      - PREFER_NOT_TO_ANSWER
+
+  GenderIdentityV2:
+    type: string
+    description: 'Which terms best express how you describe your current gender identity? (Select all that apply)'
+    enum:
+      - GENDERQUEER
+      - MAN
+      - NON_BINARY
+      - QUESTIONING           # Questioning or unsure of my gender identity
+      - TRANS_MAN             # Trans man/Transgender man
+      - TRANS_WOMAN           # Trans woman/Transgender woman
+      - TWO_SPIRIT
+      - WOMAN
+      - OTHER                 # None of these fully describe me, and I want to specify
+      - PREFER_NOT_TO_ANSWER
+
+  SexualOrientationV2:
+    type: string
+    description: 'Which terms best express how you describe your current sexual orientation? (Select all that apply)'
+    enum:
+      - ASEXUAL
+      - BISEXUAL
+      - GAY
+      - LESBIAN
+      - POLYSEXUAL            # Polysexual, omnisexual, or pansexual
+      - QUEER
+      - QUESTIONING           # Questioning or unsure of my sexual orientation
+      - SAME_GENDER           # Same-gender loving
+      - STRAIGHT              # Straight or heterosexual
+      - TWO_SPIRIT
+      - OTHER                 # None of these fully describe me, and I want to specify
+      - PREFER_NOT_TO_ANSWER
+
+  SexAtBirthV2:
+    type: string
+    description: 'What was the sex assigned to you at birth, such as on your original birth certificate?'
+    enum:
+      - FEMALE
+      - INTERSEX
+      - MALE
+      - OTHER                 # None of these fully describe me, and I want to specify
+      - PREFER_NOT_TO_ANSWER
+
+  EducationV2:
+    type: string
+    description: 'Highest Level of Education Completed'
+    enum:
+      - NO_EDUCATION          # Never attended school/No formal education
+      - GRADES_1_12           # Primary/Middle School/High School (Grades 1 through 12/GED)
+      - UNDERGRADUATE         # Some college, Associate Degree or Technical School (1 to 3 years) or current undergraduate student
+      - COLLEGE_GRADUATE      # College graduate (4 years or more) or current post-graduate trainee
+      - MASTER
+      - DOCTORATE
+      - PREFER_NOT_TO_ANSWER
+
+  YesNoPreferNot:
+    type: string
+    # note: swagger converts these to TRUE/FALSE!  https://github.com/swagger-api/swagger-parser/issues/1205
```

Comment:
TIL this is known as "The Norway Problem" https://hitchdev.com/strictyaml/why/implicit-typing-removed/

Reply:
Solved now: just need to add quotes